src/types/Cart.ts

```diff
@@ -61,9 +61,21 @@ export type CartActionRequest = {
   readonly quantity: number;
 };
 
+export type CartUpdateRequest =
+  | {
```

Comment:
What does `|` do?

Reply:
Required atleast one of them, baki sab optional hai